### ISSUE TITLE
create a dev stage

### ISSUE THREAD

**Issue by kamigerami** [codelabsab/timereport-slack#124]
Suggestion is to create a DEV stage for timereport-slack / timereport-UI / timereport-API.

If approved then the suggestion is:

# DEV
- create a dev branch
- anybody can push here without PR and it is considered unstable and could break things
- mapped to timereport-dev

# PROD
- master branch WITH  PR from DEV branch
- same as today mapped to /timereport 

this would make it easier to get things out even if it means we break things in dev.

Whenever we are happy with any changes in DEV we could just make a PR and have people look at the code before we push it to prod

**Comment by parberge** [comment]
Before we decide that we should have branches and different environments we should probably sit and try to understand what are problems are (I think we have a pretty good idea) and then we can propose proper ways to solve those problems.

**Comment by kamigerami** [comment]
As an intermediary solution - apart from discussion how we are going to solve working on this big scope (creating new slash commands or something else like mocking slack) - we have decided that anything pushed to a feature branch will go out to timereport-dev directly. This was added in #123 
by update of .travis-ci.

Keeping this ticket open until we discuss this further in the next hackaton session.
